sql/core/src/main/scala/org/apache/spark/sql/DataFrameStatFunctions.scala

```diff
@@ -85,6 +105,13 @@ final class DataFrameStatFunctions private[sql](df: DataFrame) {
    *             of the DataFrame.
    * @return A DataFrame containing for the contingency table.
    *
+   * {{{
+   *    val df = sqlContext.createDataFrame(Seq((1, 1), (1, 2), (2, 1), (2, 1), (2, 3), (3, 2),
+   *      (3, 3))).toDF(“key”, “value”)
```

Comment:
You have some UTF-8 sneaking in here which causes it to render weirdly:

![screen shot 2015-06-11 at 5 14 54 pm](https://cloud.githubusercontent.com/assets/527/8121227/9095bce6-105d-11e5-87b0-05607647a349.png)